docs/userguides/transactions.md

```diff
@@ -121,3 +121,63 @@ In your `ape-config.yaml` file, add the following:
 ```yaml
 transaction_acceptance_timeout: 600  # 5 minutes
 ```
+
+## Traces
+
+If you are using a provider that is able to fetch transaction traces, such as the [ape-hardhat](https://github.com/ApeWorX/ape-hardhat) provider, you can call the [`ReceiptAPI.show_trace()`](../methoddocs/api.html?highlight=receiptapi#ape.api.transactions.ReceiptAPI.show_trace) method.
+
+```python
+from ape import accounts, project
+
+owner = accounts.load("acct")
+contract = project.Contract.deploy(sender=owner)
+receipt = contract.methodWithoutArguments()
+receipt.show_trace()
+```
+
+The trace might look something like:
+
+```bash
+Call trace for '0xd15e59e4cca4aaf3ac57be8b71d35bb53549ec3ec1833da183d3fa5f4f609b83'
+Sender=0x1e59ce931B4CFea3fe4B875411e280e173cB7A9C
+ContractA.methodWithoutArguments() -> 0x00..7a9c [411973 gas]
+├── ContractB.methodB1(foo="bar", bar=42) [401601 gas]
+│   ├── ContractC.getSomeList() -> [
+│   │     3425311345134513461345134534531452345,
+│   │     111344445534535353,
+│   │     993453434534534534534977788884443333
+│   │   ] [369688 gas]
+│   └── ContractC.methodC1(foo="bar", baz=36, ban=ContractA) [363455 gas]
```

Comment:
As we discussed before, gas needs some work to be more meaningful. Will probably figure this out better w/ gas reporting work. (just making a note here)